mnist_test.go

```diff
@@ -47,7 +47,7 @@ func ExampleTrainMNIST() {
 	opt := torch.SGD(0.01, 0.5, 0, 0, false)
 	opt.AddParameters(net.Parameters())
 
-	epochs := 5
+	epochs := 2
```

Comment:
By reducing the number of epochs, we can save some CI time.  The benchmark outputs the correct throughput but averaged from less number of epochs.